cc_licenses/templates/base.html

```diff
@@ -20,7 +20,15 @@
   <main>
     <div class="level padding-{% start %}-big padding-{% end %}-large padding-vertical-normal">
           {% include 'includes/skiplink.html' %}
-          {% include 'includes/breadcrumbs.html' %}
+          <nav class="breadcrumb level-left caption bold" aria-label="breadcrumbs">
+            <ul>
+                <li><a href="#">{% trans "Home" %}</a></li>
+                <li><a href="#">{% trans "License" %}</a></li>
+                {% block breadcrumbs-li %}
+                  <li class="is-active"><a href="#" aria-current="page displayed">{% include "includes/snippet/license_medium_title.html" %}</a></li>
```

Comment:
It looks like - if I'm reading it all right - the only thing we're changing is the text displayed for the link. Might it make sense to make just that the block, and just have the markup for this item here?  Or maybe not - I'm not sure what this is actually supposed to link *to*, so maybe that'll eventually be changing?

Reply:
done